reference/api/settings.md

```diff
@@ -1,8 +1,6 @@
 # All settings
 
-[Index](/learn/core_concepts/indexes.md) settings are represented as a [JSON object literal](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON), containing a field for each possible customization option.
-
-It is possible to modify all the settings at once using the [`update settings` endpoint](#update-settings), or individually using the dedicated routes.
+The `/settings` route allows you to customize search settings for the given index. It is possible to modify all of an index's settings at once using the [`update settings` endpoint](#update-settings) or individually using the dedicated routes.
```

Comment:
This might be the fever speaking, but I read the second sentence as "you can modify all index settings by either using the update settings endpoint or the dedicated routes." Like, as if you could use a dedicated route to change all settings.

I would suggest slightly altering the sentence so there's no room for confusion. Perhaps breaking the sentence in two? Something like "It is possible to modify all settings […]. You can also update individual settings using the dedicated routes"?

Reply:
```suggestion
The `/settings` route allows you to customize search settings for the given index. It is possible to modify all of an index's settings at once using the [`update settings` endpoint](#update-settings), or modify each one individually using the dedicated routes.
```